docs/pages/enterprise/soc2.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: SOC2 compliance for SSH, Kubernetes and Applications
+description: How to configure SSH, Kubernetes and Application access to be SOC2 compliant
+h1: SOC2 Compliance for SSH and Kubernetes and Application Access
+---
+
+Teleport is designed to meet SOC2 requirements for the purposes of accessing infrastructure. This document outlines a high
+level overview of how Teleport can be used to help your company to become SOC2 certified.
+
+# Achieving SOC2 Compliance with Teleport
+SOC2 or Service Organization Controls were developed by the American Institute of CPAs (AICPA). They are based on five trust service areas: security, availability, processing integrity, confidentiality and privacy. Teleport helps, in part, with all five trust service areas and is a key component of how Gravitational achieved SOC2 Type 2 compliance. With Teleport, we were able to implement a Beyond Corp, Zero Trust organization that met compliance requirements without slowing developers down. 
+
+SOC Auditors look at 17 criteria (also known as principles) developed by the Committee of Sponsoring Organizations of the Treadway Commission (COSO). 
+
+Teleport is designed to help satisfy two principal areas:
+CC6: Logical and Physical Access Controls 
+CC7 System Operations 
+
+Each principle has many “Points of Focus” which will apply differently to different products and organizations, talk to an auditor to understand exactly which points of focus apply.
+
+
+
+# What Key SOC2 Controls does Teleport help achieve?
+Teleport helps with 4 of the 9 control areas. We should write in the docs some specifics of which controls.
```

Comment:
Would we want to remove the _We should write in the docs_ sentence before merging this?